benefit/data/sheets/CaseField/CaseField.json

```diff
@@ -381,6 +381,7 @@
   {"LiveFrom": "18/06/2019", "CaseTypeID": "Benefit", "ID": "hasUnprocessedAudioVideoEvidence", "Label": "Has unprocessed Audio/Video evidence", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "showWorkCapabilityAssessmentPage", "Label": "Show the Work Capability Assessment page?", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "10/06/2019", "CaseTypeID": "Benefit", "ID": "panelDoctorSpecialism", "Label": "Panel doctor specialism", "FieldType": "FixedList", "FieldTypeParameter": "FL_panelDoctorSpecialism","SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "22/03/2022", "CaseTypeID": "Benefit", "ID": "secondPanelDoctorSpecialism", "Label": "Second Panel doctor specialism ", "FieldType": "FixedList", "FieldTypeParameter": "FL_panelDoctorSpecialism","SecurityClassification": "PUBLIC"},
```

Comment:
(Minor) extra space at the end of the label

Reply:
@dattipoeHMCTS Because I follow the format from master branch. 
https://github.com/hmcts/sscs-ccd-definitions/blob/master/benefit/data/sheets/CaseField/CaseField.json#L425